src/tools.py

```diff
@@ -173,7 +182,8 @@ def b58_decode(s):
     for c in s:
         n *= 58
         if c not in b58_digits:
-            raise InvalidBase58Error('Character %r is not a valid base58 character' % c)
+            raise InvalidBase58Error('Character %r is not a ' +
```

Comment:
Personally, I didn't succeed to do the 80 char per line.
I let that on the code.  But, other `pep8` checks are good to take.
I think we can apply this rule for Silkaj project.